### DISCUSSION TITLE
Are clj-kondos linting rules a superset of jokers?

### DISCUSSION THREAD

**Discussion by port19x** [clj-kondo/clj-kondo#1974] [Q&A] [answered]
In the 2017 talk it's mentioned that clj-kondo tries to be as complete as joker with some additional linting rules even.

Has this been achieved?
Because if so, I'll deduplicate my editor setup to not lint with both, possibly saving some battery in the process

**Comment by borkdude** [comment, marked as answer]
clj-kondo does not try to be a drop-in replacement for joker, but I think by now it has most of what joker has (and much more)

**Comment by borkdude** [comment]
The talk was in 2019 btw :)
